charts/generic-prometheus-alerts/templates/application-alerts.yaml

```diff
@@ -7,8 +7,8 @@
 {{- $targetJobName := default $targetApplicationRegex .Values.jobTargetOverride }}
 {{- $targetCronjob := default $targetApplicationRegex .Values.cronjobTargetOverride }}
 {{- $targetHpa := default .Values.targetApplication .Values.hpaTargetOverride }}
-{{- $targetApplicationUnderscore := .Values.targetApplication | replace "-" "_" }}
-{{- $businessOrAllHoursExpression := ternary "and ON() {{ $targetApplicationUnderscore }}:business_hours" "" .Values.businessHoursOnly}}
```

Comment:
this wasn't expanded by helm so ended up being in prometheus query and therefore keeling over

Reply:
I should have spotted that before-  the fix looks good.